x-pack/legacy/plugins/maps/public/layers/styles/vector/components/vector_style_editor.js

```diff
@@ -264,6 +264,35 @@ export class VectorStyleEditor extends Component {
           }
         />
         <EuiSpacer size="m" />
+
+        <VectorStyleColorEditor
+          swatches={DEFAULT_LINE_COLORS}
+          onStaticStyleChange={this._onStaticStyleChange}
+          onDynamicStyleChange={this._onDynamicStyleChange}
+          styleProperty={this.props.styleProperties[VECTOR_STYLES.LABEL_BORDER_COLOR]}
+          fields={this._getOrdinalFields()}
+          defaultStaticStyleOptions={
+            this.state.defaultStaticProperties[VECTOR_STYLES.LABEL_BORDER_COLOR].options
+          }
+          defaultDynamicStyleOptions={
+            this.state.defaultDynamicProperties[VECTOR_STYLES.LABEL_BORDER_COLOR].options
+          }
+        />
+        <EuiSpacer size="m" />
+
+        <VectorStyleSizeEditor
```

Comment:
text-halo-width cannot be larger than 1/4 of the text-size. https://docs.mapbox.com/mapbox-gl-js/style-spec/#paint-symbol-text-halo-width

I think we should try to enforce this.

- (1) we can do this blackboxed. E.g in `syncLabelBorderWidthWithMb` (which could get an extra param which is the label-size).
- (2) we update the sliders. Instead of measuring 0-100 as in font-size units, what it measures  is a ratio. Than maybe 0-25% or something. Or we could retain 100, where 100 just = "largest" (which is 1/4 of the label-size) and 0 = "none".

(2) may work better too when label-sizes are data-driven.